corehq/apps/locations/tests/util.py

```diff
@@ -158,16 +171,7 @@ def tearDownClass(cls):
 
     @classmethod
     def restrict_user_to_assigned_locations(cls, user):
-        role = UserRole(
-            domain=cls.domain,
-            name='Regional Supervisor',
-            permissions=Permissions(edit_commcare_users=True,
-                                    edit_locations=True,
-                                    access_all_locations=False),
-        )
-        role.save()
-        user.set_role(cls.domain, role.get_qualified_id())
-        user.save()
+        restrict_user_by_location(cls.domain, user)
```

Comment:
`restrict_user_to_assigned_locations`/`remove_all_locations_access` sounds a better name considering we are not restricting user to any particular location but revoking `access_all_locations`?